tests/gatsby-2/test/mocha-config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  diff: true,
+  delay: false,
+  extension: ['js'],
+  package: './package.json',
+  reporter: 'spec',
+  slow: 75,
+  timeout: 2000,
+  spec: [
+    './test/**/*.test.js',
+    '../shared/**/*.test.js',
+  ],
+  require: [
+    // https://mochajs.org/#-require-module-r-module
+    // '@babel/register',
```

Comment:
If we want to write our tests in fancy ES, we can uncomment this and they'll just work without babel transpiling first.

Reply:
lol i was going to comment below about arrow functions but that's nit.